storage/azure/package.json

```diff
@@ -74,8 +74,8 @@
     "sort-package-json": "~1.53.1",
     "typescript": "~4.7.4",
     "wait-on": "~6.0.1",
-    "webpack": "~5.72.0",
-    "webpack-cli": "~4.9.2"
+    "webpack": "~5.79.0",
```

Comment:
for webpack and wepack-cli, might as well use ^ too

Reply:
I reverted the upgrade for now as it caused issues. I created an item in our backlog to upgrade these dependencies.